src/System.Text.Json/tests/Utf8JsonReaderTests.cs

```diff
@@ -1512,6 +1503,88 @@ public static void InvalidJsonWithCommentsSingleSegment(string jsonString, int e
             }
         }
 
+        [Fact]
+        public static void EmptyJsonIsInvalid()
+        {
+            var dataUtf8 = ReadOnlySpan<byte>.Empty;
+            var json = new Utf8JsonReader(dataUtf8, isFinalBlock: true, state: default);
+
+            try
+            {
+                while (json.Read())
+                    ;
+                Assert.True(false, "Expected JsonReaderException was not thrown with single-segment data.");
+            }
+            catch (JsonReaderException ex)
+            {
+                Assert.Equal(0, ex.LineNumber);
+                Assert.Equal(0, ex.BytePositionInLine);
+            }
+        }
+
+        [Fact]
```

Comment:
This needs to be a theory if you are passing a `ReadOnlySpan<byte> dataUtf8` parameter.

Looking at the test name, most likely, the test shouldn't have any arguments and the empty sequence should be passed in to the `Utf8JsonReader`